app/src/main/java/com/cmput301w23t40/capturetheqr/AddQRActivity.java

```diff
@@ -18,10 +32,51 @@ public class AddQRActivity extends AppCompatActivity {
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_addqr);
-
+        buttonSubmit = findViewById(R.id.btn_Submit);
+        buttonSubmit.setOnClickListener(v->
+        {
+            startActivity(new Intent(getApplicationContext(), MainActivity.class));
+            Toast.makeText(getApplicationContext(), R.string.add_qr_success_toast,Toast.LENGTH_LONG).show();
+        });
+//        buttonScan = findViewById(R.id.btn_scanCode);
+//        buttonScan.setOnClickListener(v->
+//        {
+//            scanCode();
+//        });
         getSupportActionBar().setDisplayHomeAsUpEnabled(true);
     }
 
+//    /**
+//     * Opens the QR code Scanner and initializes the scan options.
+//     */
+//    private void scanCode() {
+//        ScanOptions options = new ScanOptions();
+//        options.setPrompt("Scan a QR Code");
+//        options.setDesiredBarcodeFormats(ScanOptions.QR_CODE);
+//        options.setBeepEnabled(true);
+//        options.setOrientationLocked(true);
+//        options.setCaptureActivity(ScanActivity.class);
+//        barLauncher.launch(options);
+//    }
+
+//    ActivityResultLauncher<ScanOptions> barLauncher = registerForActivityResult(new ScanContract(), result ->
+//    {
+//        if(result.getContents() != null)
+//        {
+//            AlertDialog.Builder builder = new AlertDialog.Builder(AddQRActivity.this);
+//            builder.setTitle("Result");
+//            builder.setMessage(result.getContents());
+//            builder.setPositiveButton("OK", new DialogInterface.OnClickListener() {
+//                @Override
+//                public void onClick(DialogInterface dialog, int which) {
+//                    dialog.dismiss();
+//                }
+//            }).show();
+//
+//        }
+//
+//    });
```

Comment:
Is this needed? There's a scanCode in MainActivity. If not, then I think it would be better to just remove this

Reply:
@nic3-14159 I'm gonna remove those comments after I complete issue #52 and #53  , because I'm not sure yet if it's better to have the scanCode() funcitonality through the MainActivity or AddQRActivity yet, as far as implementing these features goes - so I'd prefer to either merge it with these comments or hold off on this PR until I complete those issues